cmd/zetaclientd/initconfig.go

```diff
@@ -44,18 +47,22 @@ func setupInitializeConfigOptions() {
 	f, cfg := InitializeConfigCmd.Flags(), &initializeConfigOpts
 
 	const (
-		usageMode             = "mode for cross-chain transaction processing (0:standard, 1:dry, 2:chaos)"
-		usagePeer             = "peer address e.g. /dns/tss1/tcp/6668/ipfs/16Uiu2HAmACG5DtqmQsH..."
-		usageHotKey           = "hotkey for zetaclient this key is used for TSS and ZetaClient operations"
-		usageLogLevel         = "log level (0:debug, 1:info, 2:warn, 3:error, 4:fatal, 5:panic)"
-		usageP2PDiag          = "p2p diagnostic ticker (default: 0 means no ticker)"
-		usageTicker           = "config update ticker (default: 0 means no ticker)"
-		usageKeyring          = "keyring backend to use (test, file)"
-		usageMaxBaseFee       = "the maximum base fee in Gwei allowed to send ZetaChain transactions (0 means no limit)"
-		usageMempoolThreshold = "the threshold number of unconfirmed txs in the zetacore mempool to consider it congested (0 means no threshold)"
+		usageMode                 = "mode for cross-chain transaction processing (0:standard, 1:dry, 2:chaos)"
```

Comment:
we can do it later but I think it would be ux friendlier to use a string for the option value